ldelf/trace_ext.c

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Could we avoid duplicating code? This looks VERY similar to `lib/libutee/trace_ext.c`... I guess some `#ifdef`ery would be fine.

Reply:
Okay, makes sense. Is it acceptable to extern the one `_ldelf_log()` function needed in `lib/libutee/trace_ext.c`? Or should I make the whole `ldelf/syscalls.h` header public.

Reply:
No strong opinion here, but it seems `lib/libutee/include/ldelf_syscalls.h` would be best (since we have `lib/libutee/include/utee_syscalls.h` already).

Reply:
You mean `ldelf/include/ldelf_syscalls.h`?

Reply:
Not what I meant initially ;-) but thinking about it, it sounds good. Contrary to `utee_syscalls.h`, `ldelf_syscalls.h` cannot be used by TAs but only by ldelf, so that header should not be exported in the "dev kit" and it could be confusing to have it next to `utee_syscalls.h`.

Reply:
I agree, that was my reasoning too.

Reply:
Done